src/components/OrderDetailWidget/OrderDetailWidget.tsx

```diff
@@ -118,10 +130,47 @@ const OrderDetailWidget: FC<OrderDetailWidgetProps> = ({ order }) => {
     return new Date(parseInt(order.expiry) * 1000);
   }, [order]);
 
+  const userIsFromSwapFlow = !!location.state?.fromSwapFlow;
+
   const [showFeeInfo, toggleShowFeeInfo] = useToggle(false);
+  const [showViewAllQuotes, toggleShowViewAllQuotes] = useToggle(false);
+
+  useEffect(() => {
+    if (!indexerUrls && library) {
+      dispatch(fetchIndexerUrls({ provider: library }));
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [indexerUrls, library]);
+
+  useEffect(() => {
+    if (indexerUrls && senderToken?.address && signerToken?.address) {
+      dispatch(
+        getFilteredOrders({
+          filter: {
+            senderTokens: [senderToken.address],
```

Comment:
To get orders with similar amounts we should add `senderMinAmount` and `senderMaxAmount` -25% + 25% respectively.

Reply:
i had this implemented before and we decided to take it out. if we want to do this, lets move it to another PR.